### PR TITLE
WIP: RHV-43058 Leveraging OpenShift docs contribution guidelines for oVirt

### PR DESCRIPTION
**Pull request by stoobie**
Changes proposed in this pull request:

Adding doc contribution guidelines based on OpenShift doc contribution guidelines.
I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @stoobie 

This pull request needs review by: @sandrobonazzola @duck-rh (but not yet!)

### CODE REVIEW

File: source/documentation/contributing_to_docs/contributing.adoc [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+"cherry-picked" into the release branches will show up in the published
+documentation after 12 hours.
+///
+
+All published oVirt content is on the `master` branch.
+Therefore, when submitting your work the PR must be created againstthe `master`
+branch. After it is reviewed, a writer will merge the content.
+====
+
+== Adding files to the collection
```
Comment (emarcusRH) [inline source/documentation/contributing_to_docs/contributing.adoc:141]: This section needs to be commented out or removed - only applies to OpenShift repo

**Comment by sandrobonazzola** [comment]
@emarcusRH can you take over this PR as @stoobie is not working on this anymore?

**Comment by apinnick** [comment]
@sandrobonazzola 
The OpenShift-doc contributor guidelines are for downstream technical writers. OpenShift-doc does not have upstream writers or guidelines. They are not appropriate for an upstream project. The fact that the docs are in Asciidoc will also be a barrier. You might want to consider eventually converting them to markdown.

When would you need contributor guidelines?

**Comment by sandrobonazzola** [comment]
@apinnick I am not sure about why @stoobie was working on this, if you don't see this fitting oVirt I would be fine just closing this PR.

**Pull request closed without merging**